docs/releasing.md

```diff
@@ -19,13 +19,15 @@ $ npx mynewcli
 
 You'll need to [register with npm](https://www.npmjs.com/signup) and have verified your email address in order to publish.
 
-This workflow can be improved slightly by running `npm version major|minor|patch` before publishing which will create a git tag and bump the version automatically. We like to use [np](https://npm.im/np) which is like `npm version`, but will also run the tests and other health checks before publishing.
+This workflow can be improved slightly by running `npm version major|minor|patch` before publishing which will create a git tag and bump the version automatically.
 
 ## Standalone tarballs
 
-Build standalone tarballs with `oclif pack` from the root of your CLI. These include the node binary so the user does not have to already have node installed to use the CLI. It won't put this node binary on the PATH so the binary will not conflict with any node installation on the machine.
+Build standalone tarballs with `oclif pack tarballs` from the root of your CLI. These include the node binary so the user does not have to already have node installed to use the CLI. It won't put this node binary on the PATH so the binary will not conflict with any node installation on the machine.
 
-To publish, you can copy the files from `./dist` or use `oclif promote` to copy the files to S3. You'll need to set `oclif.update.s3.bucket` in `package.json` to a valid S3 bucket and have credentials set in `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment vars.
+To publish, you can copy the files from `./dist` or use `oclif upload tarballs` to copy the files to S3. You'll need to set `oclif.update.s3.bucket` in `package.json` to a valid S3 bucket and have credentials set in `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment vars.
+
+After you've uploaded the tarballs to S3, you can promote the tarballs to a different release channel within S3 using `oclif promote`. This allows you to quickly promote and demote a specific version between your various release channels. For example, the Salesforce CLI has separate `stable` and `stable-rc` channels that are updated weekly.
```

Comment:
`oclif upload tarballs` doesn't create a release channel. It just uploads the tarballs to a directory derived from its intended target.
```suggestion
After you've uploaded the tarballs to S3, you can promote the tarballs to a release channel within S3 using `oclif promote`. This allows you to quickly promote and demote a specific version between release channels. For example, the Salesforce CLI has separate `stable` and `stable-rc` channels that are updated weekly.
```

Reply:
Should these docs have a reference to Salesforce CLI as an example?

Reply:
@peternhale I don't see why not. It's just an example. We could reference other oclif based CLIs if you have any suggestions.